zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/MutableChecksumsSFV.java

```diff
@@ -40,6 +40,14 @@ public interface MutableChecksumsSFV extends ImmutableChecksumsSFV {
    */
   void updateFromSfvFile(final String... lines);
 
+  /**
+   * Update the checksum map with the given checksum
+   *
+   * @param filePath path to file
+   * @param checksum check of file given
+   */
+  void updateFromChecksum(final Path filePath, final Long checksum);
```

Comment:
:upside_down_face: 

```suggestion
  void updateFromChecksum(final Path filePath, final long checksum);
```

Always a bit nicer to use the primitive types instead of the boxed ones.